app/services/remediations_api.rb

```diff
@@ -9,20 +9,27 @@ def initialize(account)
   end
 
   def import_remediations
-    Rule.find_in_batches(batch_size: 100).each do |rules|
-      response = Platform.connection.post(@url) do |req|
-        req.headers['X-RH-IDENTITY'] = @b64_identity
-        req.headers['Content-Type'] = 'application/json'
-        req.body = { 'issues': build_issues_list(rules) }.to_json
-      end
-      update_rules(remediations_available(response))
+    rules_with_profiles.each_slice(100) do |rules|
+      update_rules(remediations_available(remediations_request(rules)))
     end
   rescue Faraday::ClientError => e
     Rails.logger.info("#{e.message} #{e.response}")
   end
 
   private
 
+  def remediations_request(rules)
```

Comment:
Weird that the method is named `...request` when it actually returns a response.